mappings/net/minecraft/world/gen/DensityFunctions.mapping

```diff
@@ -0,0 +1,445 @@
+			ARG 1 o
+		METHOD m_czahebwa fromValue ()D
+		METHOD m_khgjowhl fromY ()I
+		METHOD m_mitcetii toY ()I
+		METHOD m_nearmskl toValue ()D
+	CLASS C_dsnmcorv BeardifierOrMarker
+		FIELD f_zrnhfgsw CODEC Lcom/mojang/serialization/Codec;
+	CLASS C_ihcxcncj MultiplyOrAdd
+		FIELD f_axuyyfjb argument D
+		FIELD f_bbjjjkwr operationType Lnet/minecraft/unmapped/C_mdkmcezi$C_ihcxcncj$C_oasfbkwq;
+		FIELD f_lotenpvv input Lnet/minecraft/unmapped/C_kzwapyud;
+		FIELD f_ltmwezav maxValue D
+		FIELD f_nfrfpmxe minValue D
+		METHOD <init> (Lnet/minecraft/unmapped/C_mdkmcezi$C_ihcxcncj$C_oasfbkwq;Lnet/minecraft/unmapped/C_kzwapyud;DDD)V
+			ARG 1 operationType
+			ARG 2 input
+			ARG 3 minValue
+			ARG 5 maxValue
+			ARG 7 argument
+		METHOD equals (Ljava/lang/Object;)Z
+			ARG 1 o
+		METHOD m_acrcrutr argument ()D
+		METHOD m_saupsjal operationType ()Lnet/minecraft/unmapped/C_mdkmcezi$C_ihcxcncj$C_oasfbkwq;
+		CLASS C_oasfbkwq Type
+	CLASS C_iixtwqxj Slide
+		FIELD f_jkrmeymm settings Lnet/minecraft/unmapped/C_ulvgadxi;
+		FIELD f_uzkeupev input Lnet/minecraft/unmapped/C_kzwapyud;
+		FIELD f_xyufqzlz CODEC Lcom/mojang/serialization/Codec;
+		METHOD <init> (Lnet/minecraft/unmapped/C_ulvgadxi;Lnet/minecraft/unmapped/C_kzwapyud;)V
+			ARG 1 settings
+			ARG 2 input
+		METHOD equals (Ljava/lang/Object;)Z
+			ARG 1 o
+		METHOD m_xbkspnqv settings ()Lnet/minecraft/unmapped/C_ulvgadxi;
+	CLASS C_jqsjvrgl Shift
+		FIELD f_pupingrd CODEC Lcom/mojang/serialization/Codec;
+		FIELD f_sgackxpt offsetNoise Lnet/minecraft/unmapped/C_rxrtfjrt;
+		FIELD f_spdmecjl noiseData Lnet/minecraft/unmapped/C_cjzoxshv;
+		METHOD <init> (Lnet/minecraft/unmapped/C_cjzoxshv;Lnet/minecraft/unmapped/C_rxrtfjrt;)V
+			ARG 1 noiseData
+			ARG 2 offsetNoise
+		METHOD equals (Ljava/lang/Object;)Z
+			ARG 1 o
+	CLASS C_jufcfxbf Ap2
+		FIELD f_dfwvotuo type Lnet/minecraft/unmapped/C_mdkmcezi$C_xtxhyvmj$C_aittffza;
+		FIELD f_mitbfgpw maxValue D
+		FIELD f_qfosausu argument1 Lnet/minecraft/unmapped/C_kzwapyud;
+		FIELD f_tcqwhdkt minValue D
+		FIELD f_wplqsdsm argument2 Lnet/minecraft/unmapped/C_kzwapyud;
+		METHOD <init> (Lnet/minecraft/unmapped/C_mdkmcezi$C_xtxhyvmj$C_aittffza;Lnet/minecraft/unmapped/C_kzwapyud;Lnet/minecraft/unmapped/C_kzwapyud;DD)V
+			ARG 1 type
+			ARG 2 argument1
+			ARG 3 argument2
+			ARG 4 minValue
+			ARG 6 maxValue
+		METHOD equals (Ljava/lang/Object;)Z
+			ARG 1 o
+	CLASS C_lhdfeucz EndIslandDensityFunction
+		FIELD f_fmwzwyui islandNoise Lnet/minecraft/unmapped/C_mnmvtskp;
+		FIELD f_kemeiodl CODEC Lcom/mojang/serialization/Codec;
+		METHOD <init> (J)V
+			ARG 1 random
+	CLASS C_lrntbvay BlendDensity
+		FIELD f_lwmifaxw input Lnet/minecraft/unmapped/C_kzwapyud;
+		FIELD f_mhhtjgoe CODEC Lcom/mojang/serialization/Codec;
+		METHOD <init> (Lnet/minecraft/unmapped/C_kzwapyud;)V
+			ARG 1 input
+		METHOD equals (Ljava/lang/Object;)Z
+			ARG 1 o
+	CLASS C_mcdsqker BlendAlpha
+		FIELD f_yjjnpjin CODEC Lcom/mojang/serialization/Codec;
+	CLASS C_mefvoqiu HolderHolder
```

Comment:
...well, okay.

Reply:
I mean, that's what it would do, so it made sense